pkg/repository/manifest.go

```diff
@@ -280,6 +424,20 @@ func readTimestampManifest(input io.Reader, keys *KeyStore) (*Timestamp, error)
 	return &ts, nil
 }
 
+// BatchSaveManifests write a series of manifests to disk
+func BatchSaveManifests(dst string, manifestList []ValidManifest) error {
+	for _, m := range manifestList {
+		writer, err := os.OpenFile(filepath.Join(dst, m.Filename()), os.O_WRONLY|os.O_CREATE, 0644)
+		if err != nil {
+			return err
+		}
+		if err = signAndWrite(writer, m); err != nil {
```

Comment:
I have never seen there is a snippet to close the writer.

Reply:
I'm no expert on this, I would think that all files need closing. In this case there is a `close` in the example of https://golang.org/pkg/os/#OpenFile, but it does not use defer.